ApplicationLibCode/ProjectDataModel/Seismic/RimSeismicSection.cpp

```diff
@@ -953,40 +954,67 @@ void RimSeismicSection::setDepthFilter( RimIntersectionFilterEnum filterType, in
     m_zLowerThreshold = lowerValue;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+int RimSeismicSection::alignZValue( int z ) const
+{
+    if ( m_seismicData == nullptr ) return z;
+
+    const int zMin  = (int)m_seismicData->zMin();
+    const int zStep = (int)m_seismicData->zStep();
+
+    int alignedZ = ( ( z - zMin ) / zStep );
+    alignedZ *= zStep;
+    alignedZ += zMin;
+
+    return alignedZ;
+}
+
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
 int RimSeismicSection::upperFilterZ( int upperGridLimit ) const
 {
+    int retVal;
```

Comment:
Missing variable initialization

Reply:
no need, one of two options below always is selected.